pkg/controller/dgraph/models/service.go

```diff
@@ -54,8 +54,24 @@ func StoreService(service api_v1.Service) error {
 			Name:      service.Name,
 			IsService: true,
 			ID:        dgraph.ID{Xid: xid},
+			StartTime: service.GetCreationTimestamp().Time,
 		}
 		bytes := utils.JSONMarshal(newService)
+		assigned, err := dgraph.MutateNode(bytes)
+		if err != nil {
+			return err
+		}
+		uid = assigned.Uids["blank-0"]
```

Comment:
If we are consistently using `assigned.Uids["blank-0"]` for the uid, we can probably update `dgraph.MutateNode(bytes)` to directly return `uid` instead of `assigned`.

Reply:
@gurusreekanthc will output `assigned` be useful in other ways? May be when we do batch update?

Reply:
we need uid in case of create, for delete it's not needed.